tests/apm/mod.rs

```diff
@@ -170,5 +180,5 @@ fn logging() {
 
     coll.drop().unwrap();
 
-    fs::remove_file("test_apm_log.txt").unwrap();
+    fs::remove_file("test_log.txt").unwrap();
```

Comment:
I've noticed/started to get annoyed that the test file doesn't get cleaned up when the tests fail above this line, so I'm thinking we should change the tests so this file doesn't clutter things up. Given that the file should only be about a dozen lines, I think reading the whole thing into a string and then iterating over the lines should be fine.